src/skip-to-content/skip-to-content.scss

```diff
@@ -15,23 +15,26 @@
   position: fixed;
   text-align: center;
   left: 50%;
+  opacity: 0;
   top: $iui-baseline * 2;
   transform: translateX(-50%) translateY(-170%);
   transition: background-color $iui-speed-fast ease-in-out;
   @media (prefers-reduced-motion: no-preference) {
-    transition: background-color $iui-speed-fast ease-in-out, transform $iui-speed-slow ease-in-out,
-      box-shadow $iui-speed-fast ease-in-out;
+    transition: opacity $iui-speed-slow ease-in-out, background-color $iui-speed-fast ease-in-out,
```

Comment:
why this weird mix of `slow` and `fast`?

Reply:
Everything within the media query is now matching slow.  This is so when the button loses `:focus` it slowly fades away off the screen.
The one `transition` outside of the media query can be set to fast for the button hover effect.  Tested with reduced motion on to make sure everything is working as expected.